controller/iteration_blackbox_test.go

```diff
@@ -720,7 +710,7 @@ func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space
 		rootIterationObj = iteration.Iteration{
 			Name:       spaceObj.Name,
 			SpaceID:    spaceObj.ID,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c